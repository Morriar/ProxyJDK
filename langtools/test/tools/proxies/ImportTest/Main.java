/*
 * @test
 * @summary generic types tests
 * @author Alexandre Terrasa
 *
 * @compile Package/Main.java RealSubject/RunnableRealSubject.java Main.java
 */

package TMPProxyTests.ImportTest;

import TMPProxyTests.ImportTest.RealSubject.RunnableRealSubject;

public class Main {
	public static void main(String[] args) {
		Runnable prxy = proxy Runnable(new RunnableRealSubject("TestImport.TestImport"));
		prxy.run();
	}
}
