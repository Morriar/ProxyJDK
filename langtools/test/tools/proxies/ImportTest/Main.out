run by TestImport.TestImport
run by TestImport2